core/mastodon/src/main/java/br/com/orcinus/orca/core/mastodon/feed/profile/account/MastodonAccount.kt

```diff
@@ -101,13 +104,23 @@ internal data class MastodonAccount(
     avatarLoaderProvider: SomeImageLoaderProvider<URI>,
     postPaginatorProvider: MastodonProfilePostPaginator.Provider
   ): Profile {
-    return if (isOwner()) {
+    return if (isOwned()) {
       toEditableProfile(context, requester, avatarLoaderProvider, postPaginatorProvider)
     } else {
       toFollowableProfile(context, requester, avatarLoaderProvider, postPaginatorProvider)
     }
   }
 
+  /**
+   * Whether the currently authenticated [Actor] is the owner of this [Account].
+   *
+   * @param authenticationLock [AuthenticationLock] through which the ID of the [Actor] will be
+   *   compared to that of this [MastodonAccount].
+   */
+  suspend fun isOwned(authenticationLock: SomeAuthenticationLock): Boolean {
```

Comment:
Add an `isOwnedAsync(SomeAuthenticationLock): CompletionStage<Boolean>` and obtain the value within a coroutine scope in each Kotlin-specific overload?

Such change would then allow for removing [`CompletableContinuation`](https://github.com/orcinusbr/orca-android/blob/0a401dbe53bbc3e45d07ca9c11565d02418a1841/core/mastodon/src/main/java/br/com/orcinus/orca/core/mastodon/notification/interop/CompletableContinuation.java) and [its utility class](https://github.com/orcinusbr/orca-android/blob/0a401dbe53bbc3e45d07ca9c11565d02418a1841/core/mastodon/src/main/java/br/com/orcinus/orca/core/mastodon/notification/interop/CompletableContinuations.kt), which exist only for enabling interoperability between Java synchronicity and Kotlin suspension.